scripts/ci/circle/run.sh

```diff
@@ -8,8 +9,9 @@ REALBRANCH="${CIRCLE_BRANCH}"
 if [ -n "${CIRCLE_PR_NUMBER}" ]
 then
   APIURL="${API_BASE}/pulls/${CIRCLE_PR_NUMBER}"
-  RESULT="$(curl -s ${APIURL} | python3 -c "import sys, json; print(json.load(sys.stdin)['head']['ref'])" 2> /dev/null)"
-  if [ $? -eq 0 ]
+  RESULT="$(curl -s "${APIURL}" | python3 -c "import sys, json; print(json.load(sys.stdin)['head']['ref'])" 2> /dev/null)"
+  exit_status=$?
+  if [ "$exit_status" -eq 0 ];
```

Comment:
```suggestion
  if [ "$exit_status" -eq 0 ]
```

conditionals in bash does not need a semicolon if the `then` is in the next line

Apply this change in for the below cases too